lib/Interpreter/ValueExtractionSynthesizer.cpp

```diff
@@ -395,34 +408,54 @@ namespace {
     return Call.get();
   }
 
-  void ValueExtractionSynthesizer::FindAndCacheRuntimeDecls() {
+  static bool VSError(clang::Sema* Sema, clang::Expr* E, llvm::StringRef Err) {
```

Comment:
Well - but why introduce VSError here then if a simple "don't suppress diagnostics" suffices? With the change proposed here, we first disable the default diagnostics to then emit our custom ones.

If this is a sequencing / merge conflict etc issue then I am not against merging this first, as long as you promise by the honor of all kangaroos, wallabies, koalas, possums, opossums, wombats and Tasmanian devils to hand that PR in at some later point! :-)